src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -259,6 +268,10 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
     /* hash */ resourceLoader.bootConfig.resources.runtime[dotnetWasmResourceName],
     /* type */ 'dotnetwasm'
   );
+  const totalResources = assembliesBeingLoaded.concat(pdbsBeingLoaded, wasmBeingLoaded);
+  assembliesBeingLoaded.forEach(loadingResource => loadingResource.response.then(_ => setProgress()));
+  pdbsBeingLoaded.forEach(loadingResource => loadingResource.response.then(_ => setProgress()));
+  wasmBeingLoaded.response.then(_ => setProgress());
```

Comment:
We may be able to simplify the call here somewhat. 

```suggestion
  const totalResources = assembliesBeingLoaded.concat(pdbsBeingLoaded, wasmBeingLoaded);
```

and then after adding the optional `timezone` and `icudata`, you can do:

```js
  totalResources.forEach(loadingResource => loadingResource.response.then(_ => setProgress()));
```

Technically, you could map the resources from `LoadingResource -> LoadingResource.Response` thereby only having the actual promises in the `totalResources`, but not sure how others feel about that additional "work".